src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle implements Figure {
+    private String color;
+    private double radius;
+
+    public Circle(String color, double radius) {
+        this.color = color;
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, area: "
+                + getArea()
+                + " sq. units, radius: "
+                + radius
+                + " units, color: "
+                + color);
```

Comment:
Use `name()` instead of `toString()` for getting String representation of enum constants. If `color` is an enum, you should use `color.name()`.